client/src/app/api/models.ts

```diff
@@ -190,18 +190,17 @@ export type IdentityKind =
 
 export interface Identity {
```

Comment:
we need to sync the API manually - it would be nice to have a link (as a comment)  to the original backend structure i.e. https://github.com/konveyor/tackle2-hub/blob/94330e1ff9cdb479db3b6aeb30f9abb6859c16bc/api/identity.go#L229

Reply:
Maybe a comment with a link to where the openapi.json is kept?

Reply:
> There would be lots of comments if we started doing that
>
right. let's discuss this and provide some global solution as a separate issue/PR